pkg/operator/controllers/banner/banner.go

```diff
@@ -0,0 +1,48 @@
+package banner
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	BannerName         = "openshift-aro-sre"
+	TextContactSupport = "Please ask your cluster administrator to contact Azure or Red Hat support. Your cluster's resourceid: %s"
+)
+
+func (r *BannerReconciler) reconcileBanner(ctx context.Context, instance *arov1alpha1.Cluster) error {
+	var err error
```

Comment:
Probably makes more sense to use a named return value here: I imagine it will be less error prone as it will be harder overshadow it in some cases.

Reply:
Fixed.